tests/integration_test.rs

```diff
@@ -0,0 +1,107 @@
+            <http://blank.distribution#0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/dcat#Distribution> .
+        "#,
+    ).await;
+}
+
+#[tokio::test]
+async fn inline_distribution() {
+    assert_transformation(
+        r#"
+            @prefix dcat: <http://www.w3.org/ns/dcat#> .
+            @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+            _:00 rdf:type dcat:Dataset ;
+            dcat:distribution [ rdf:type dcat:Distribution ] .
+        "#,
+        r#"
+            <http://blank.dataset#0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/dcat#Dataset> .
+            <http://blank.dataset#0> <http://www.w3.org/ns/dcat#distribution> <http://blank.distribution#0> .
+            <http://blank.distribution#0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/dcat#Distribution> .
+        "#,
+    ).await;
+}
+
+#[tokio::test]
+async fn real_world_example() {
+    assert_transformation(
+        r#"
+            @prefix dcat: <http://www.w3.org/ns/dcat#> .
+            @prefix dct: <http://purl.org/dc/terms/> .
+            @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+            _:00 rdf:type dcat:Dataset ;
+            dct:description "dataset desc"@nb ;
+            dcat:distribution [ rdf:type dcat:Distribution ;
+            dct:description "dist desc"@nb ;
+            dct:format <https://www.iana.org/assignments/media-types/application/vnd.openxmlformats-officedocument.spreadsheetml.sheet> , <https://www.iana.org/assignments/media-types/text/csv> ;
+            dct:license <http://data.norge.no/nlod/no/2.0> ;
+            dct:title "foo"@nb ] .
+        "#,
+        r#"
+            <http://blank.dataset#0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/dcat#Dataset> .
+            <http://blank.dataset#0> <http://purl.org/dc/terms/description> "dataset desc"@nb .
+            <http://blank.dataset#0> <http://www.w3.org/ns/dcat#distribution> <http://blank.distribution#0> .
+            <http://blank.distribution#0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/dcat#Distribution> .
+            <http://blank.distribution#0> <http://purl.org/dc/terms/title> "foo"@nb .
+            <http://blank.distribution#0> <http://purl.org/dc/terms/format> <https://www.iana.org/assignments/media-types/text/csv> .
+            <http://blank.distribution#0> <http://purl.org/dc/terms/format> <https://www.iana.org/assignments/media-types/application/vnd.openxmlformats-officedocument.spreadsheetml.sheet> .
+            <http://blank.distribution#0> <http://purl.org/dc/terms/description> "dist desc"@nb .
+            <http://blank.distribution#0> <http://purl.org/dc/terms/license> <http://data.norge.no/nlod/no/2.0> .
+        "#,
+    ).await;
+}
+
+async fn assert_transformation(input: &str, output: &str) {
```

Comment:
`expected` blir vel et tydeligere navn her
```suggestion
async fn assert_transformation(input: &str, expected: &str) {
```

Reply:
Nice!

https://github.com/Informasjonsforvaltning/fdk-mqa-node-namer/pull/6/commits/b3b19b20f7cf09758fe1d636a6b2e86cafc16eb9